qiling/core.py

```diff
@@ -125,8 +123,8 @@ def __init__(
         self.global_thread_id = 0
         self.gdb = None
         self.gdbsession = None
-        self.automatize_input = automatize_input
-        self.config = config
+        self.automatize_input = False
+        self.config = os.path.dirname(os.path.abspath(__file__)) + "/os/windows/configuration.cfg"
```

Comment:
you might need a os.path.join here. to avoid "\" for windows and "/" for linux

Reply:
You are absolutely right.